### PR TITLE
feat: update campaigns, restructure, cleanup

### PR DESCRIPTION
**Pull request by Ming-Yan**
- Fix undefined parameter errors in github actions
- Add DeepJet binning info to `definitions.py`

### CODE REVIEW

**Review by andrzejnovak — CHANGES_REQUESTED** [review]
There are some straggling lines left over and please run black (https://github.com/psf/black) but otherwise LGTM

File: src/BTVNanoCommissioning/workflows/__init__.py [hunk 1]
```diff
@@ -66,11 +66,19 @@
 workflows["ctag_ttsemilep_sf"] = CTAGSemilepttValidSFProcessor
 workflows["ectag_ttsemilep_sf"] = CTAGETTSemilepValidSFProcessor
 
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (Ming-Yan) [reply src/BTVNanoCommissioning/workflows/__init__.py]: fix

File: README.md [hunk 2]
```diff
@@ -36,6 +36,11 @@ conda activate btv_nano_commissioning
 ```
 You could simply create the environment through the existing `env.yml` under your conda environment
 ```
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (Ming-Yan) [reply README.md]: fix

File: README.md [hunk 3]
```diff
@@ -32,10 +32,12 @@ You can either use the default environment `base` or create a new one:
 # create new environment with python 3.7, e.g. environment of name `coffea`
 conda create --name btv_nano_commissioning python=3.7
 # activate environment `coffea`
+```
```
Comment (andrzejnovak) [inline README.md:35]: These are wrong I think

**Review by andrzejnovak — APPROVED** [review]

**Pull request merged**